packages/world/ts/test/entity.test.ts

```diff
@@ -69,3 +75,112 @@ describe("EntityId encoding", () => {
     );
   });
 });
+
+describe("EntityId decoding", () => {
+  it("decodeBlock", () => {
+    expect(
+      decodePosition(
+        "0x0300000000000000000000000000000000000000000000000000000000000000",
+      ),
+    ).toEqual([0, 0, 0]);
+    expect(
+      decodePosition(
+        "0x03000000270000003e0000001000000000000000000000000000000000000000",
+      ),
+    ).toEqual([39, 62, 16]);
+    expect(
+      decodePosition(
+        "0x03fffffff4000000000000000300000000000000000000000000000000000000",
+      ),
+    ).toEqual([-12, 0, 3]);
+    expect(
+      decodePosition(
+        "0x03ffffff9afffffffe00000be400000000000000000000000000000000000000",
+      ),
+    ).toEqual([-102, -2, 3044]);
+    expect(
+      decodePosition(
+        "0x03fffffc16fffffffefffff41c00000000000000000000000000000000000000",
+      ),
+    ).toEqual([-1002, -2, -3044]);
+  });
+  it("decodeFragment", () => {
+    expect(
+      decodePosition(
+        "0x0200000000000000000000000000000000000000000000000000000000000000",
+      ),
+    ).toEqual([0, 0, 0]);
+    expect(
+      decodePosition(
+        "0x02000000270000003e0000001000000000000000000000000000000000000000",
+      ),
+    ).toEqual([39, 62, 16]);
+    expect(
+      decodePosition(
+        "0x02fffffff4000000000000000300000000000000000000000000000000000000",
+      ),
+    ).toEqual([-12, 0, 3]);
+    expect(
+      decodePosition(
+        "0x02ffffff9afffffffe00000be400000000000000000000000000000000000000",
+      ),
+    ).toEqual([-102, -2, 3044]);
+    expect(
+      decodePosition(
+        "0x02fffffc16fffffffefffff41c00000000000000000000000000000000000000",
+      ),
+    ).toEqual([-1002, -2, -3044]);
+  });
+  it("decodePlayer", () => {
+    expect(
+      decodePlayer(
+        "0x01f39fd6e51aad88f6f4ce6ab8827279cfffb922660000000000000000000000",
+      ),
+    ).toEqual("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266");
+    expect(
+      decodePlayer(
+        "0x0170997970c51812dc3a010c7d01b50e0d17dc79c80000000000000000000000",
+      ),
+    ).toEqual("0x70997970C51812dc3A010C7d01b50e0d17dc79C8");
+    expect(
+      decodePlayer(
+        "0x013c44cdddb6a900fa2b585dd299e03d12fa4293bc0000000000000000000000",
+      ),
+    ).toEqual("0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC");
+    expect(
+      decodePlayer(
+        "0x0190f79bf6eb2c4f870365e785982e1f101e93b9060000000000000000000000",
```

Comment:
is there any strong reason to left vs right align the address within the entity ID?

```
0x0190f79bf6eb2c4f870365e785982e1f101e93b9060000000000000000000000
```
```
0x01000000000000000000000090f79bf6eb2c4f870365e785982e1f101e93b906
```

Reply:
nope no reason